packages/jbrowse-web/src/plugins/BigWigAdapter/BigWigAdapter.js

```diff
@@ -27,6 +55,48 @@ export default class BigWigAdapter extends BaseAdapter {
     return header.totalSummary
   }
 
+  // todo: add caching
+  // todo: incorporate summary blocks
+  async getRegionStats({ refName, start, end }) {
```

Comment:
@garrettjstevens we should make sure the stats-fetching is included in #192 adapter capabilities